.github/workflows/ci.yml

```diff
@@ -69,3 +73,84 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - run: cargo doc
+
+  # just cargo check for now
+  matrix_test:
+    runs-on: ${{ matrix.platform.os }}
+    if: github.event.pull_request.draft == false
+    env:
+      RUST_CARGO_COMMAND: ${{ matrix.platform.cross == true && 'cross' || 'cargo' }}
+    strategy:
+      matrix:
+        platform:
+          - {
+              target: x86_64-pc-windows-msvc,
+              os: windows-latest,
+              toolchain: "1.75.0",
```

Comment:
Why `1.75`?

Reply:
Also, does `toolchain` need to be specified per-platform. Could just make global?